source/common/upstream/subset_lb.h

```diff
@@ -177,8 +198,14 @@ class SubsetLoadBalancer : public LoadBalancer, Logger::Loggable<Logger::Id::ups
   LbSubsetEntryPtr fallback_subset_;
   LbSubsetEntryPtr panic_mode_subset_;
 
+  absl::optional<LbSubsetEntryPtr> selector_fallback_subset_any_;
```

Comment:
It is probably overkill to use optional with a pointer value as nullptr pointer value is typically used to indicate that the pointer points to "nothing".